src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -640,8 +646,23 @@ public void postRunnable(Runnable item) {
         
         @TargetApi(Build.VERSION_CODES.LOLLIPOP)
         @Override
-        public void onReceivedClientCertRequest (WebView view, final ClientCertRequest request){
-            Logger.v(TAG + "onReceivedClientCertRequest", "onReceivedClientCertRequest");
+        public void onReceivedClientCertRequest (WebView view, final ClientCertRequest request)
+        {
+            final String methodName = ":onReceivedClientCertRequest";
+            Logger.v(TAG + methodName, "Webview receives client TLS request.");
```

Comment:
It might be better to grab the calling method name from a stack trace in the logger (http://stackoverflow.com/questions/5439406/java-equivalent-to-func) and not have to worry about copy/paste errors.